src/test/java/com/digitalpartners/test/LoginTest.java

```diff
@@ -40,6 +40,6 @@ public void verifyBookCalender() throws InterruptedException {
     public void tearDown() {
         getDriver().quit();
 		System.out.println("Hello");
-		System.out.println("Hiiiiiiiiiii");
+		System.out.println("Sweet Dream");

```

Comment:
This is not correct java coding slandered. Please Use Sweet instead of Sweet Dream.

Reply:
Resolved.